.eslintrc.js

```diff
@@ -49,10 +49,10 @@ module.exports = {
     '@typescript-eslint/restrict-template-expressions': 0,
 
     // VV These rules we'd like to turn off one day so they error.
-    '@typescript-eslint/no-unsafe-assignment': 'warn',
-    '@typescript-eslint/no-unsafe-member-access': 'warn',
-    '@typescript-eslint/no-unsafe-call': 'warn',
-    '@typescript-eslint/no-unsafe-return': 'warn'
+    '@typescript-eslint/no-unsafe-assignment': 0,
+    '@typescript-eslint/no-unsafe-member-access': 0,
+    '@typescript-eslint/no-unsafe-call': 0,
+    '@typescript-eslint/no-unsafe-return': 0
```

Comment:
Are these settings on by default? Maybe we can just remove them.

Reply:
Let's keep them as a reminder that we would like to get rid of these cases in code at some pint.